python/cuml/dask/ensemble/base.py

```diff
@@ -42,6 +43,7 @@ def _create_model(self, model_func,
         self.client = get_client(client)
         self.workers = self.client.scheduler_info()['workers'].keys()

```

Comment:
Interesting that create_model takes a `workers` parameter but then queries the client for the workers instead?

Reply:
Yeah, it seemed redundant might make sense to remove the workers from the creat_model and just pass the client in

Reply:
My bad, it seems like the user is allowed to choose a subset of workers they want to work with when creating an RF model. So, what we do in `base` would be incorrect if we use all of the workers available. 

@Salonijain27 Do you have know why `workers` is being passed and then ignored for later computations?

Reply:
Leaving it as it is now, might be worth coming back after the release.

Reply:
@Nanthini10  this should be updated to :
```
        if workers is None:
            workers = self.client.scheduler_info()['workers'].keys()  # Default to all workers
        self.workers = workers
```

Reply:
Thanks!